src/services/submissionService.ts

```diff
@@ -45,6 +44,18 @@ export async function getSubmissions(filters: dbSchema.SubmissionSearchFilters =
             subRef = subRef.where('isRejected', '==', filters.isRejected)
         }
 
+        if (filters.orderBy && Array.isArray(filters.orderBy)) {
```

Comment:
a best practice for queries is to have a default sort order and limit in place. 
This should use it if the user sends one in, but we should set a default here of order by descending `createdDate`  (latest ones first) and limit to 20. 

Sorry I didn't think of this sooner! 
I noticed that we don't have a createdDate or updatedDate which we should likely add to everything we save.